src/SDKs/Kusto/Management.Kusto/Microsoft.Azure.Management.Kusto.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Kusto</PackageId>
+    <Description>Provides developers with libraries for the updated kusto platform under Azure Resource manager to create ane manage clusters. Note: This client library is for Virtual Machines under Azure Resource Manager.</Description>
+    <Version>1.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.Kusto</AssemblyName>
+    <PackageTags>management;kusto;;;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is a public preview release of the Azure Kusto SDK. Included with this release is creating and managing clusters.
+      ]]>
+</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
```

Comment:
Please remove 20, 21, `AssemblyVersion` and `AssemblyFileVersion` must be specified in the `AssemblyInfo.cs` file